frappe/www/login.html

```diff
@@ -66,12 +66,12 @@ <h6>{{ _("Or login with") }}</h6>
 	<div class="login-content page-card" style="margin-top: 20px;">
 	<form class="form-signin form-signup hide" role="form">
 		<div class="page-card-head">
-			<span class="indicator blue" data-text="{{ _("Sign Up") }}"></span>
+			<span class="indicator blue" data-text="{{ _('Sign up') }}"></span>
```

Comment:
@binardev Please keep it in Title Case to make it consistent since we are aiming to have Labels and Button's text to be in Title Case.
https://github.com/frappe/erpnext/wiki/Naming-Guidelines#field-naming
```suggestion
			<span class="indicator blue" data-text="{{ _('Sign Up') }}"></span>
```
You can contribute translation for `_('Sign Up')`

Follow: https://frappe.io/docs/user/en/guides/basics/contribute_translations